packages/asc-ui/src/utils/themeUtils.ts

```diff
@@ -150,12 +140,13 @@ export const svgFill = (
   if (colorType) {
     const value = color(colorType, variant)({ theme })
     if (typeof value === 'string') {
-      return `
+      return `& > svg {
```

Comment:
absolutely sure you changed this in all the component that use this util?

Reply:
Yes. Checked!